contracts/peg/PegStabilityModuleNonCustodial.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {IPCVDeposit} from "./../pcv/IPCVDeposit.sol";
+import {FixedPricePSM} from "./FixedPricePSM.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/// @notice Peg Stability Module that holds no funds.
+/// On a mint, it transfers all proceeds to a PCV Deposit and then deposits them into the deposit's target
+/// When funds are needed for a redemption, they are simply pulled from the PCV Deposit
+contract PegStabilityModuleNonCustodial is FixedPricePSM {
```

Comment:
Why is this is a fixed price psm?

Reply:
Hmm, I don't agree with this layering of abstractions. 

It feels like a `NonCustodial` PSM is a much more general abstraction than a `FixedPricePSM`. The `NonCustodial` can be used, and if we add more PSMs likely will be, across asset type; whereas the `FixedPricePSM` is an asset specific abstraction. 

So, it makes more sense imo for `NonCustodial` to be the 'base' and have the `FixedPricePSM` inherit from it. 

I'm not a fan of how nested our PSM contracts are becoming and how long the inheritance chain is becoming, but there's not much we can do about that here

Reply:
Just inherit from the regular PSM

Reply:
Done